tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -142,9 +148,26 @@ def test_node_maintenance(
         assert typed_nodes, f"Failed to find a {node_type} node for the test"
         typed_node_name = typed_nodes[0].name
 
+        # check csi-cephfsplugin-provisioner's and csi-rbdplugin-provisioner's
+        # are ready, see BZ #2162504
+        provis_pods = get_pods_having_label(
+            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
+            defaults.ROOK_CLUSTER_NAMESPACE,
+        )
+        provis_pods += get_pods_having_label(
+            constants.CSI_RBDPLUGIN_PROVISIONER_LABEL,
+            defaults.ROOK_CLUSTER_NAMESPACE,
+        )
+        provis_pod_names = [p["metadata"]["name"] for p in provis_pods]
```

Comment:
Here you can use `p.name` instead of `p["metadata"]["name"]`.

Reply:
here we operate pod items (list of dictionaries) but not actual pods (more at get_pods_having_label), so we'll get AttributeError in case of p.name. We need only pods names for `wait_for_pods_to_be_running` so I don't create list of Pods here (resource consuming)

Reply:
Okay, got it.